src/docs/patterns/appbar.mdx

```diff
@@ -8,6 +8,8 @@ import Image4 from '../../app/assets/design-patterns/search-bar/search-bar-searc
 
 App bars are useful in a variety of locations throughout PX Blue applications. They are typically used to provide a title or description for a collection of data that follows. They may also contain a menu and/or various actions that can be performed.
 
+<MaterialDesignDescription title={'App bars: top'} url={'https://material.io/components/app-bars-top/'} />
```

Comment:
```suggestion
<MaterialDesignDescription title={'App Bars: Top'} url={'https://material.io/components/app-bars-top/'} />
```

Reply:
I know how you have it is how the title is shown on Material, but for me it feels a little weird on doc-it...all of our headlines are in Title Case.